qa/L0_multiple_ports/test.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+# Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+#
+# Redistribution and use in source and binary forms, with or without
+# modification, are permitted provided that the following conditions
+# are met:
+#  * Redistributions of source code must retain the above copyright
+#    notice, this list of conditions and the following disclaimer.
+#  * Redistributions in binary form must reproduce the above copyright
+#    notice, this list of conditions and the following disclaimer in the
+#    documentation and/or other materials provided with the distribution.
+#  * Neither the name of NVIDIA CORPORATION nor the names of its
+#    contributors may be used to endorse or promote products derived
+#    from this software without specific prior written permission.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS ``AS IS'' AND ANY
+# EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+# IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR
+# PURPOSE ARE DISCLAIMED.  IN NO EVENT SHALL THE COPYRIGHT OWNER OR
+# CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO,
+# PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR
+# PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY
+# OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+# (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+
+# MULTI_PORT_CLIENT=../clients/multi_port_client
+MULTI_PORT_CLIENT_PY=../clients/multi_port_client.py
+
+CLIENT_LOG="./client.log"
+
+DATADIR=`pwd`/models
+HSP_ARR=(8008 8009 8010 8011)
+HHP_ARR=(8008 8010 8010 8009)
+HPP_ARR=(8008 8011 8011 8010)
+HIP_ARR=(8008 8011 8008 8008)
+SERVER=/opt/tensorrtserver/bin/trtserver
+for (( n=0; n<4; n++ ))
+do
+:
+# do whatever on $i
+  SERVER_ARGS_ADD_HTTP="--http-status-port ${HSP_ARR[n]} --http-health-port ${HHP_ARR[n]} --http-profile-port ${HPP_ARR[n]}\
```

Comment:
We need to discuss the cases that need testing. It seems you are missing a lot of important ones. Also, since the last arg is "--http-port -1" (and "--grpc-port -1") then aren't all the ports disabled and so the server shouldn't even start?